content/docs/hello-world.md

```diff
@@ -1,50 +1,75 @@
 ---
 id: hello-world
-title: Hello World
+title: Witaj, świecie!
 permalink: docs/hello-world.html
 prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Poniżej znajdziesz przykład najprostszej aplikacji reactowej:
 
 ```js
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>Witaj, świecie!</h1>,
   document.getElementById('root')
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Wyświetli ona na stronie nagłówek "Witaj, świecie".
 
-[](codepen://hello-world)
+**[Spróbuj w CodePen](codepen://hello-world)**
```

Comment:
Na Slacku rozmawialiśmy z @jakubdrozdek, że dobrze byłoby trzymać tę nazwę spójną, jako `Przetestuj kod na CodePen`? Często będzie się to powtarzać, więc warto trzymać się jednej konwe
```suggestion
**[Przetestuj kod na CodePen](codepen://hello-world)**
```

Reply:
Racja, te linki powinny mieć jednakowy tekst.